content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -0,0 +1,3253 @@
+
+#### Reference
+
+```ts
+distinct?: Enumerable<UserDistinctFieldEnum>
+```
+
+```ts
+export declare const UserDistinctFieldEnum: {
+  id: 'id'
+  name: 'name'
+  email: 'email'
+  profileViews: 'profileViews'
+  role: 'role'
+  coinflips: 'coinflips'
+}
+```
+
+#### Examples
+
+##### Select distinct on a single field
+
+The following example returns all distinct `city` fields, and selects only the `city` and `country` fields:
+
+<CodeWithResult  expanded={true}>
+
+<Cmd>
+
+```ts
+const distinctCities = await prisma.user.findMany({
+  select: {
+    city: true,
+    country: true,
+  },
+  distinct: ['city'],
+})
+```
+
+</Cmd>
+
+<CmdResult>
+
+```js no-lines
+;[
+  { city: 'Paris', country: 'France' },
+  { city: 'Lyon', country: 'France' },
+]
+```
+
+</CmdResult>
+
+</CodeWithResult>
+
+##### Select distinct on multiple fields
+
+The following example returns all distinct `city` _and_ `country` field combinations, and selects only the `city` and `country` fields:
+
+<CodeWithResult expanded={true}>
+
+<Cmd>
+
+```ts
+const distinctCitiesAndCountries = await prisma.user.findMany({
+  select: {
+    city: true,
+    country: true,
+  },
+  distinct: ['city', 'country'],
+})
+```
+
+</Cmd>
+
+<CmdResult>
+
+```js no-lines
+;[
+  { city: 'Paris', country: 'France' },
+  { city: 'Paris', country: 'Denmark' },
+  { city: 'Lyon', country: 'France' },
+]
+```
+
+</CmdResult>
+
+</CodeWithResult>
+
+Note that there is now a "Paris, Denmark" in addition to "Paris, France":
+
+##### Select distinct in combination with a filter
+
+The following example returns all distinct `city` _and_ `country` field combinations where the user's email contains `"prisma.io"`, and selects only the `city` and `country` fields:
+
+<CodeWithResult>
+
+<Cmd>
+
+```ts
+const distinctCitiesAndCountries = await prisma.user.findMany({
+  where: {
+    email: {
+      contains: 'prisma.io',
+    },
+  },
+  select: {
+    city: true,
+    country: true,
+  },
+  distinct: ['city', 'country'],
+})
+```
+
+</Cmd>
+
+<CmdResult>
+
+```js
+;[
+  { city: 'Paris', country: 'Denmark' },
+  { city: 'Lyon', country: 'France' },
+]
+```
+
+</CmdResult>
+
+</CodeWithResult>
+
+## Nested queries
+
+### <inlinecode>create</inlinecode>
+
+A nested `create` query adds a new related record or set of records to a parent record. See: [Working with relations](../../../concepts/components/prisma-client/relation-queries#add-or-remove-related-records) <span class="concept"></span>.
+
+#### Remarks
+
+- `create` is available as a nested query when you `create` (`prisma.user.create(...)`) a new parent record or `update` (`prisma.user.update(...)`) an existing parent record.
+
+#### Examples
+
+##### Create a new `User` record with a new `Profile` record
+
+```ts
+const user = await prisma.user.create({
+  data: {
+    email: 'alice@prisma.io',
+    profile: {
+|     create: { bio: 'Hello World' },
```

Comment:
Seems like this needs an extra space

```suggestion
|      create: { bio: 'Hello World' },
```

![image](https://user-images.githubusercontent.com/4058327/105838472-69ee6700-5fd0-11eb-8612-0261be7038fa.png)